adagrams/game.py

```diff
@@ -1,11 +1,95 @@
+import random
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+SCORE_CHART = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2: ["D", "G"], 
+    3: ["B", "C", "M", "P"], 
+    4: ["F", "H", "V", "W", "Y"], 
+    5: ["K"], 
+    8: ["J", "X"], 
+    10: ["Q", "Z"]
+    }
```

Comment:
I like your decision to make `SCORE_CHART` a dictionary where the keys are the numbers!